examples/discord_bot.py

```diff
@@ -52,18 +52,47 @@ async def player_heroes(ctx, player_tag):
 async def clan_info(ctx, clan_tag):
     clan = await coc_client.get_clan(clan_tag)
 
+    for a in clan.members:
```

Comment:
It would be good to utilise the helper function, 
```suggestion
    leader = clan.get_member_by(role=coc.Role.leader)
```